sklearn/tests/test_impute.py

```diff
@@ -535,15 +535,26 @@ def test_iterative_imputer_zero_iters():
 
 
 def test_iterative_imputer_verbose():
-    n = 10
+    rng = np.random.RandomState(0)
+
+    n = 100
     d = 3
-    X = sparse_random_matrix(n, d, density=0.10).toarray()
+    X = sparse_random_matrix(n, d, density=0.10, random_state=rng).toarray()
     imputer = IterativeImputer(missing_values=0, max_iter=1, verbose=1)
```

Comment:
could you put verbose at 2. I think that we have a case that we want `verbose > 1`

Reply:
I did this two lines down (545):

```
imputer.verbose = 2
imputer.transform(X)
```

Reply:
Nevermind, I'll just do it twice.